main/plugins/org.talend.hadoop.distribution/src/main/java/messages.properties

```
@@ -18,3 +18,7 @@ EDatabriksCloudProvider.AWS=AWS
 EDatabriksCloudProvider.Azure=Azure
 EDatabriksSubmitMode.CREATE_RUN_JOB=Create and run now
 EDatabriksSubmitMode.RUN_SUBMIT=Runs submit
+ESparkMode.YARN_CLUSTER=Yarn cluster
+ESparkMode.KUBERNETES=Kubernetes
+ESparkMode.SPARK_LOCAL=Local

```

```suggestion
ESparkMode.LOCAL=Local

```
Having SPARK_ prefix seems redundant but not a big deal if it has too large impact

---

thats the way its defined in tSparkConf so it needs to be the same to avoid confusion, the UI text may be different tho